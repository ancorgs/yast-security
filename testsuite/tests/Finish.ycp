/*
 * YaST2: Modules testsuite
 *
 * Description:
 *   Testsuite for the security module
 *
 * Authors:
 *   Michal Svec <msvec@suse.cz>
 *
 * $Id$
 */

{

include "testsuite.ycp";
include "security/io.ycp";

TEST(``(SecurityFinish()),[],nil);

}
