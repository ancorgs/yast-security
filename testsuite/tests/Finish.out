Write	.etc.login_defs nil true
Write	.rc.system nil true
Execute	.target.bash "/sbin/SuSEconfig --nonewpackage" 0
Return	[true, true, 0]
